styles/designs/webview/parts/_exercise-components.scss

```diff
@@ -49,23 +49,16 @@ $Exercise__ProblemSolution__Number: (
   _name: "ProblemSolutionNumber",
   _subselector: ' > .os-number',
   _properties: (
-    display: table-cell,
-  )
-);
-
-$Exercise__ProblemSolution__Divider:(
```

Comment:
It looks like you got rid of the component styling for `$Exercise__ProblemSolution__Divider`, but didn't remove the references to the component from the webview shapes (in styles/designs/webview/parts/_exercise-shapes.scss) -- this will crash if you try to build styles as is.

Reply:
Thank you for the guidance. I'm encouraged to know that I was mostly on the right track.
I have rebased the branch so that the commits are orderly.